src/modules/analytics/sagas.ts

```diff
@@ -231,3 +233,7 @@ function* handleDeployToWorldSuccess(action: DeployToWorldSuccessAction) {
 
   yield call(track, '[Success] Deploy to World', { project_id: project.id, eth_address: ethAddress, subdomain: deployment.world })
 }
+
+function* handleSetENSAddressSuccess(action: SetENSAddressSuccessAction) {
+  yield call(track, 'Map Address to Name', { name: action.payload.ens.name, address: action.payload.address })
```

Comment:
Would you mind moving this track code to the `/src/modules/analytics/track.ts` file if possible?